app/assets/javascripts/lit/lit_frontend.js

```diff
@@ -0,0 +1,89 @@
+//= require ./mousetrap.js
+
+var $btn, $elem;
+
+buildLocalizationForm = function(e){
+  $this = $(this)
```

Comment:
You forgot about semicolon at the end of the line.

You should use `var` when declaring variables. Without `var` variables become global variables, which can lead to some unexpected problems.

You could use coffeescript and forget about all semicolon and `var`s. It also wraps whole script in a closure so `buildLocalizationForm` (and other declared functions) would also not leak into the global scope.

---

If you don't want to use coffeescript for some reason then you should at least consider adding "use strict" directive (https://www.w3schools.com/js/js_strict.asp).